plugins/scaffolder-backend/sample-templates/bitbucket-demo/template.yaml

```diff
@@ -20,7 +20,15 @@ spec:
           ui:options:
             allowedHosts:
               - bitbucket.org
-              - server.bitbucket.com
+            allowedOwners:
```

Comment:
I'm not sure of the best place, probably a comment or some documentation file, but I feel like we should mention somewhere that these lists are optional, and what happens when you don't provide them. What do you think?